File: moneylog_spring/src/main/java/bitc/fullstack503/moneylog_spring/dto/IncomeLogDTO.java
package bitc.fullstack503.moneylog_spring.dto;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.Data;
@Data
public class IncomeLogDTO
{
  @JsonProperty ("incomeLogSeq")
  private int incomeLogSeq;
  @JsonProperty ("memberId")
  private String memberId;
  @JsonProperty ("incomeMoney")
  private int incomeMoney;
  @JsonProperty ("incomeDate")
  private String incomeDate;
  @JsonProperty ("incomeCate")
  private String incomeCate;
  @JsonProperty ("incomeMemo")
  private String incomeMemo;
  @JsonProperty ("incomeSource")
  private String incomeSource;
  @JsonProperty ("incomeUse")
  private String incomeUse;
}

File: moneylog_app/app/src/main/java/bitc/example/app/dto/IncomeLogDTO.kt
package bitc.example.app.dto

import com.google.gson.annotations.SerializedName
import java.time.LocalDateTime

data class IncomeLogDTO(
  @SerializedName("incomeLogSeq")
  var incomeLogSeq: Int? = null,
  @SerializedName("memberId")
  var memberId: String? = null,
  @SerializedName("incomeMoney")
  var incomeMoney: Int? = null,
  @SerializedName("incomeDate")
  var incomeDate: String? = null,
  @SerializedName("incomeCate")
  var incomeCate: String? = null,
  @SerializedName("incomeMemo")
  var incomeMemo: String? = null,
  @SerializedName("incomeSource")
  var incomeSource: String? = null,
  @SerializedName("incomeUse")
  var incomeUse: String? = null
)
